connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -213,7 +213,8 @@ class SparkSession private[sql] (
    * @param sqlText
    *   A SQL statement with named parameters to execute.
    * @param args
-   *   A map of parameter names to literal values.
+   *   A map of parameter names to string values that are parsed as SQL literal expressions. For
+   *   example, map keys: "abc", "def_1", map values: "1", "'abc'", "DATE'2023-03-21'".
```

Comment:
How about keys that clarify the type, at least implicitly. Also arity didn't match up
```suggestion
   *   example, map keys: "rank", "name", "birthdate"; map values: "1", "'Steven'", "DATE'2023-03-21'".
```